docs/DeveloperGuide.adoc

```diff
@@ -447,6 +447,73 @@ In other words, when RestaurantBook#deleteMember() or RestaurantBook#deleteIngre
+
+* **Alternative 2:** Hide the displayed database to show the statistics.
+** Pros: There would be only 1 window to manage at any point in time.
+** Cons: Cannot view the statistics and the database concurrently.
+
+* Decision: Alternative 1 was adopted as it is an easier implementation. Implementing the `stats` window as a static window that will not change would not affect the other parts of the app, and there is no need to hide and show the current screen. Moreover, the user might want to view the statistics together with the database, which is only possible with multiple windows.
+
+=== The panel UI for `Member`, `Staff`, `Booking`, and `Ingredient`
+
+The restaurant book displays the entire database in the form of "cards", which is the same as AB4. Each item has its own panel.
+
+=== Current implementation
+
+The panel is formatted in the `ItemListPanel.fxml` file, and each item has its own controller (`MemberListPanel`, `StaffListPanel`, `BookingListPanel`, and `IngredientListPanel`) that extends from `ItemListPanel`. The panel displays the database via cards. Each item has its own card (`MemberCard`, `StaffCard`, `BookingCard`, and `IngredientCard`).
+
+==== Design Considerations
+
+===== Aspect: Design of card display
+
+* **Alternative 1 (current choice):** Making a separate fxml file for each item card.
+** Pros: Able to customise the card to suit the item.
+** Cons: Need to duplicate common elements that all the cards share, such as the numbering on the list.
+
+* **Alternative 2:** Using the same fxml file for each item card.
+** Pros: Less duplication.
+** Cons: Hard to customise the contents of each card.
+
+* Decision: Alternative 1 was adopted as each card do not share much in common other than the numbering on the list. Each card needs to display different information. For example, the `Member` item would need to display the email, while the `Ingredient` item would need to display other fields such as the quantity. Hence, not much code is duplicated, and creating 4 files would be the easiest implementation.
+
+===== Aspect: Design of panel display
+
+* **Alternative 1 (current choice):** Using the same fxml file for each item panel.
+** Pros: Less duplication.
+** Cons: Hard to customise the contents of each panel. 
+
+* **Alternative 2:** Making a separate fxml file for each item panel
+** Pros: Able to customise the panel to suit the item.
+** Cons: Need to duplicate common elements that all the item panels share.
+
+* Decision: Alternative 1 was adopted. Currently, the only difference between the panels are the header which contains the name of the item type. This only affects a single label, which is easy to edit via code. Hence, making them share the fxml reduces the code duplication. If the display of the panels were to be modified in the future, say to add more padding or to change the size, only one fxml file would need to be modified.
```

Comment:
So is the header a label?

making them share the fxml -> sharing the fxml
You really need to kill unnecessary words